OsmAnd/src/net/osmand/plus/helpers/ImportHelper.java

```diff
@@ -164,7 +164,7 @@ public void handleFileImport(final Uri intentUri, final String fileName, final b
 
 	private String getNameFromContentUri(Uri contentUri) {
 		final String name;
-		final Cursor returnCursor = app.getContentResolver().query(contentUri, null, null, null, null);
+		final Cursor returnCursor = app.getContentResolver().query(contentUri, new String[] {OpenableColumns.DISPLAY_NAME}, null, null, null);
```

Comment:
I think it is a good solution. We do not need to now all the columns here. We only need one column. It seems to be more reliable too, because ```ContentResolver.query(uri, null, null, null, null)``` in some cases return null while ```ContentResolver().query(contentUri, new String[] {OpenableColumns.DISPLAY_NAME}, null, null, null)``` does not return null in the same case.